.github/workflows/workflow.yml

```diff
@@ -363,7 +363,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python_version: [3.6, 3.7, 3.8, 3.9]
+        python_version: [3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
Why quote ?

Reply:
@5A11 otherwise it becomes a float number 3.1
it's a common issue